Core/src/org/sleuthkit/autopsy/timeline/TimeLineTopComponent.java

```diff
@@ -75,6 +79,21 @@
 
     private final TimeLineController controller;
 
+    /**
+     * Listener that drives the ContentViewer when in List ViewMode.
+     */
```

Comment:
I think it would be a bit more self-documenting if the InvalidationListener was named selectedEventedListener instead of just selectionListener.